Automattic/crowdsignal-forms
# Use standard button for popup trigger

This PR

  - removes the media upload section for the trigger button and related code
  - makes the trigger back into a button
  - apply styles to trigger button

Preview:
![new-trigger](https://user-images.githubusercontent.com/157240/116483979-8b285300-a85e-11eb-9485-9f36f17e3240.gif)


## Test instructions
Checkout and `make client`. Insert a Feedback block on a post.

On the Feedback block sidebar "Feedback button" section, there should be no media upload option, only the shadow toggle.

The trigger should be an editable button, customize the text and confirm on public frontend

~~Note: so far the button is a standard `isPrimary` button, this will probable change when we add the "edit button styling" feature~~

NOTE 2: a *cascade* of styles is applied to our buttons, we might want to improve this in the future. Right now the button is wrapped with `wp-block-button`, inherits from `wp-block-button__link` and is customized with `crowdsignal-forms-feedback__trigger` and `crowdsignal-forms-feedback__feedback-button`
